pytket/extensions/qujax/qujax_convert.py

```diff
@@ -28,104 +28,58 @@ def _tk_qubits_to_inds(tk_qubits: Sequence[Qubit]) -> Tuple[int, ...]:
-        if gate_name == "Barrier":
-            continue
-        gate_name_seq.append(gate_name)
-        qubit_inds_seq.append(_tk_qubits_to_inds(c.qubits))
-        n_params = len(c.op.params)
-        param_inds_seq.append(jnp.arange(param_index, param_index + n_params))
-        param_index += n_params
-
-    return gate_name_seq, qubit_inds_seq, param_inds_seq, circuit.n_qubits
-
-
-def tk_to_qujax(circuit: Circuit) -> qujax.UnionCallableOptionalArray:
-    """
-    Converts a pytket circuit into a function that maps circuit parameters
-    to a statetensor. Assumes all circuit gates can be found in qujax.gates.
-    Parameter argument of created function will be ordered as in circuit.get_commands()
-    - pytket automatically reorders some gates, consider using Barriers.
-
-    :param circuit: Circuit to be converted.
-    :type circuit: pytket.Circuit
-    :return: Function which maps parameters (and optional statetensor_in)
-        to a statetensor.
-        If the circuit has no parameters, the resulting function
-        will only take the optional statetensor_in as an argument.
-    :rtype: Callable[[jnp.ndarray, jnp.ndarray = None], jnp.ndarray]
-        or Callable[[jnp.ndarray = None], jnp.ndarray]
-        if no parameters found in circuit
-    """
-
-    return qujax.get_params_to_statetensor_func(*tk_to_qujax_args(circuit))
-
-
-def tk_to_qujax_args_symbolic(
-    circuit: Circuit, symbol_map: Optional[dict] = None
-) -> Tuple[Sequence[str], Sequence[Sequence[int]], Sequence[Sequence[int]], int]:
-    """
-    Converts a pytket circuit with symbolics parameters and a symbolic parameter map
-    into a tuple of arguments representing a qujax quantum circuit.
-    Assumes all circuit gates can be found in qujax.gates.
-    Note that the behaviour of tk_to_qujax_args_symbolic(circuit)
-    is different to tk_to_qujax_args(circuit),
-    tk_to_qujax_args_symbolic will look for parameters in circuit.free_symbols()
-    and if there are none it will assume that none of the gates require parameters.
-    On the other hand, tk_to_qujax_args will work out which gates are parameterised
-    based on e.g. circuit.get_commands()[0].op.params
-
-    :param circuit: Circuit to be converted.
-    :type circuit: pytket.Circuit
-    :param symbol_map: dict that maps elements of circuit.free_symbols() (sympy)
-        to parameter indices.
-    :type symbol_map: Optional[dict]
-    :return: Tuple of arguments defining a (parameterised) quantum circuit
-        that can be sent to qujax.get_params_to_statetensor_func.
-    :rtype: tuple
-        gate_name_seq: Sequence of gate name strings to be found in qujax.gates.
-        qubit_inds_seq: Sequence of sequences describing which qubits gates act on.
-        param_inds_seq: Sequence of sequences parameter indices that gates are using.
+        that can be sent to qujax.get_params_to_statetensor_func.\n
+        gate_name_seq: Sequence of gate name strings to be found in qujax.gates.\n
```

Comment:
What are the names here referring to? The elements of a tuple are not named?

Reply:
should be fixed now (unnamed)